tests/unit/web/test_views.py

```diff
@@ -32,7 +33,7 @@ def setUp(self):
         self.data = {
             'name': 'New Message',
             'email': 'newuser@test.com',
-            'message': 'New Message for Conatct'
+            'message': 'New Message for Contact'
```

Comment:
You can open a new PR for this typo fix. Please remove this from this PR since its not a part of this.